.evergreen/csfle/activate-kmstlsvenv.sh

```diff
@@ -44,6 +44,7 @@ activate_kmstlsvenv() {
       "boto3~=1.19.0"
       "git+https://github.com/kevinAlbs/PyKMIP.git@DRIVERS-2659" # Add work around for DRIVERS-2659
       "sqlalchemy<2.0.0" # sqlalchemy.exc.InvalidRequestError: Implicitly combining column managed_objects.uid with column crypto_objects.uid under attribute 'unique_identifier'.
+      "flask>=3.0"
```

Comment:
Instead of Flask, consider using the built-in `http.server`. This may enable testing network errors.

Here is an example: https://gist.github.com/kevinAlbs/3b3b84359bb0e350bf494597bcff0167

It also adds a `/set_failpoint` endpoint to tell the server to fail the next decrypt request. That may enable driver tests to more easily control how/when the test is expected to fail.